sdk/streamlib-python-wheel/tests/device_exchange_probes.py

```
@@ -356,16 +362,147 @@ def _probe(self, ctx: RuntimeContextFullAccess) -> dict:
+                # step. A wrong layout reads garbage, not FILL_CONSTANT_RGBA.
+                import torch
+
+                resolved_texture.lock()
+                device_view = torch.from_dlpack(resolved_texture)
+                observation["opaque_device_pixel"] = (
+                    device_view[11, 13].to("cpu").tolist()
+                )
+                del device_view
+                resolved_texture.unlock()
+                try:
+                    resolved_texture.bytes_per_row
```

Ruff (0.16.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)